gcloud/storage/blob.py

```diff
@@ -358,7 +402,8 @@ def _check_response_error(request, http_response):
             raise make_exception(faux_response, http_response.content,
                                  error_info=request.url)
 
-    def upload_from_file(self, file_obj, rewind=False, size=None,
+    # pylint: disable=too-many-locals
```

Comment:
OK. So for now put `# pylint: enable=too-many-locals` directly after that method (same indent level as the disable statement).

Then look inside the method and see if there is any behavior that can be factored out. Too many locals is usually a sign that we are "doing too much"